pytest_inmanta/test_parameter/boolean_parameter.py

```diff
@@ -51,7 +52,7 @@ def __init__(
         environment_variable: str,
         usage: str,
         *,
-        default=False,
+        default: Optional[Union[bool, DynamicDefault[bool]]] = False,
```

Comment:
Is this some leftover from before creating the new type?

Reply:
No, why?

Reply:
No, why?

Reply:
Nevermind